### DISCUSSION TITLE
Netlify auth is useless

### DISCUSSION THREAD

**Discussion by mannubhai1** [community/community#120450] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

My Netlify registration is not getting done.
That stupid website says my GitHub is suspicious, so it asked me for my identity proof.
I tried my passport, institute id card, aadhar card, but the fucking website can't even process them properly.
Cant register myself in netlify.

**Comment by ladunjexa** [comment]
Hi @mannubhai1,

If you haven't created deployments on Netlify or Vercel yet, consider completely deleting your accounts on those platforms. Additionally, visit the following link: https://github.com/settings/installations/. Look for Vercel and Netlify, then click on Configure. In the danger zone, click on Uninstall`.

After that:

For Vercel: Visit the following link: https://vercel.com/signup. For the free version, click on 'Hobby'. Then, choose your preferred username for your Vercel account and click on 'Continue with GitHub'.

For Netlify: Visit the following link: https://app.netlify.com/signup and click 'Sign Up with GitHub'. Follow the steps provided by the site.

If you're unsure how to delete your Netlify account, click here: [Netlify Deletion Guide Link](https://answers.netlify.com/t/deletion-of-netlify-account/106651). For Vercel, click here: [Vercel Deletion Guide Link](https://vercel.com/guides/how-do-i-delete-my-vercel-account).

Perform these steps from a browser that is already logged into your GitHub account.

If this response serves you well, please mark it as an answer :)
